contents/handbook/growth/customer-support.md

```diff
@@ -20,6 +20,14 @@ These are the ways in which customers can currently reach us:
 - **Community questions** - users can also search [previously answered questions](/questions) that have been asked anywhere on posthog.com in our Docs. This is a great way to help us improve our Docs for simpler use-case type questions, but more complex questions should be re-routed via a support ticket. 
 - **Dedicated Slack channels** - For higher-paying (or potential higher-paying) customers, we offer a dedicated channel on our main company Slack.
 
+Sometimes people reach out to us on Twitter/X. If this happens, you should:
+
+1. Check first if they already have a ticket in Zendesk (either in-app or via /questions). There is nothing more annoying for a user than being asked to create a support ticket if they already have. 
+2. If no, ask them to create a support ticket within the app - this is _much_ better than trying to solve their problem over Twitter as Zendesk pulls in a bunch of contextual information and is easier to collaborate in. Do this from the **PostHog** Twitter account - otherwise you will get personally contacted every time this user wants help. 
+3. If yes, nudge CS even if it's a low priority ticket - CS are responsible for following up. Let the user know that you've done this. Again, use the PostHog Twitter account.
+
+Generally with Twitter, your objective should be to get the conversation into Zendesk asap, because it's easier to help the person there. An exception to this rule is if you are an engineer engaging with someone who has provided general feedback about PostHog - feel free to use your PostHog account if someone has a feature request or similar. 
```

Comment:
My worry here is that this basically gives users an incentive to nag individuals on Twitter, so that their tickets get escalated. I think it's better if we do NOT let users know. Point 1 also requires that people check Zendesk, which not everyone has access to. 

The whole thing will be much simpler if the policy is


```suggestion
Sometimes people reach out to us with support issues on Twitter/X. Regardless of whether someone reaches out to your personal account or to the company account the broad approach should be as follows:

1. **Tell the user clearly that we can't provide support over social media.** Instead, ask them to either create a support ticket in-app or post in /questions so that a ticket is created. Assure the user all tickets are triaged and will be responded to. 

2. **Move the conversation to a private channel if the user continues to engage.** If the user is engaging via your personal account, stop responding and reply only through the official PostHog account. If you don't have access to the official Twitter account, post in the Team-Customer-Success channel so someone else can take over.

3. **Highlight the issue with CS via Slack.** CS are responsible for following up. Do not make promises to the user about how fast we'll respond or that you've escalated the ticket. Thank them for highlighting the problem and assure them that CS will look into it and respond. 

Your objective should be to get the conversation into Zendesk ASAP, to limit further public discussion, and to avoid setting a precedent that complaining visibly on social media results in expedited responses. 

If users engage in a way which causes you _any_ discomfort or distress, disengage and highlight the issue with the CS team via Slack. 
```

Reply:
1. I don't think people are going to track down this page and abuse this, but appreciate the language leaves that possibility open. 
2. Checking whether a ticket exists first is critical to not annoying users further - if you don't have access to Zendesk, you should ask someone to check for you. 

I'll edit my original wording a bit, but I think you've gone too far in the other direction of doing what is good for PostHog only.

Reply:
(unless you think it's simpler to say "never engage" here and let people step on social toes if they think it's the right to do)

Reply:
I've updated if you'd like to take a read again